plugins/guests/linux/cap/mount_smb_shared_folder.rb

```diff
@@ -34,10 +35,17 @@ def self.mount_smb_shared_folder(machine, name, guestpath, options)
           mnt_opts << "credentials=/etc/smb_creds_#{name}"
           mnt_opts << "uid=#{mount_uid}"
           mnt_opts << "gid=#{mount_gid}"
+          if !ENV['VAGRANT_DISABLE_SMBMFSYMLINKS']
+            mnt_opts << "mfsymlinks"
+          end
 
           mnt_opts = merge_mount_options(mnt_opts, options[:mount_options] || [])
 
           mount_options = "-o #{mnt_opts.join(",")}"
+          if mount_options.include? "mfsymlinks"
```

Comment:
Would you please add parens on the method: `mount_options.include?("mfsymlinks")`